include/revng/RestructureCFG/ScopeGraphGraphTraits.h

```diff
@@ -173,4 +312,14 @@ debug_function inline void dumpScopeGraph(llvm::Function &F) {
   for (auto *DFSNode : llvm::depth_first(Scope<llvm::Function *>(&F))) {
     llvm::dbgs() << DFSNode->getName().str() << "\n";
   }
+
+  // Print the dominator tree
+  llvm::DomTreeOnView<llvm::BasicBlock, Scope> DominatorTree;
+  DominatorTree.recalculate(F);
+  DominatorTree.print(llvm::dbgs());
+
+  // Print the postdominato tree
```

Comment:
Typo: `postdominato`

Reply:
Fixed this in https://github.com/revng/revng/blob/51973f8d187490659acf4c2a6c0dbad558d0a9b3/include/revng/RestructureCFG/ScopeGraphGraphTraits.h#L321